### PR TITLE
Create blitzllama-cohort.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/blitzllama-cohort.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+--8<-- "includes/editions-all-paid-editions.md"
+
+[Blitzllama](https://www.linkedin.com/company/blitzllama/) GPT-powered feedback analytics automatically categorizes survey responses into topics, sentiments, and provides recommendations. This saves time that would be spent manually summarizing the feedback. As all the feedback is stored in one place, you can segment historical feedback and create charts within a few clicks.
+
+This integration lets you sync cohorts from Amplitude to Blitzllama.
+
+!!!info
+
+    This integration is maintained by  Blitzllama. Contact the Blitzllama support team at tech@blitzllama.com for support with this integration.
```
Comment (bgiori) [inline docs/data/destinations/blitzllama-cohort.md:14]:
```suggestion
    This integration is maintained by  Blitzllama. Contact the Blitzllama support team at [tech@blitzllama.com](mailto:tech@blitzllama.com) for support with this integration.
```

File: docs/data/destinations/blitzllama-cohort.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+2. Click on Amplitude under the Sources section.
+3. Click **Connect** and copy the API key.
+
+
+### Amplitude setup
+
+1. In Amplitude Data, click **Catalog** and select the **Destinations** tab.
+2. In the Cohort section, click **Blitzllama**.
+3. Enter Name and paste in the API key you copied from **Blitzllama**.
+4. Map the same Amplitude User_ID with the primary key from the Blitzllama panel.
```
Comment (bgiori) [inline docs/data/destinations/blitzllama-cohort.md:38]:
```suggestion
4. Map the same Amplitude User ID with the primary key from the Blitzllama panel.
```

**Review by bgiori — COMMENTED** [review]
Screen Shot 2023-05-30 at 9 35 47 AM

can we ask the blitzllama team for a better icon, one without a big black box border?

**Review by Leoaqr — APPROVED** [review]

**Comment by kevinpagtakhan** [comment]
@Brandoncharleskhoo1 do we still need these PRs? if so, can we fix the merge conflicts? appears to have been approved and just waiting to get merged

File: docs/data/destinations/blitzllama-cohort.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+title: Send Cohorts to Blitzllama
+description: Send Amplitude cohorts to Blitzllama.
+---
+
+--8<-- "includes/editions-all-paid-editions.md"
+
+[Blitzllama](https://www.linkedin.com/company/blitzllama/) GPT-powered feedback analytics automatically categorizes survey responses into topics, and sentiments, and provides recommendations. This saves time that would be spent manually summarizing the feedback. As all the feedback is stored in one place, you can segment historical feedback and create charts within a few clicks.
```
Comment (markzegarelli) [inline docs/data/destinations/blitzllama-cohort.md]: @Brandoncharleskhoo1 Any reason we're linking to LInkedIn here, and not their website?

File: docs/data/destinations/blitzllama-cohort.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+---
+title: Send Cohorts to Blitzllama
+description: Send Amplitude cohorts to Blitzllama.
+---
+
+--8<-- "includes/editions-all-paid-editions.md"
+
+[Blitzllama](https://www.linkedin.com/company/blitzllama/) GPT-powered feedback analytics automatically categorizes survey responses into topics, and sentiments, and provides recommendations. This saves time that you would spend manually summarizing the feedback. Blitzllama stores feedback in one place, you can segment historical feedback and create charts within a few clicks.
```
Comment (markzegarelli) [inline docs/data/destinations/blitzllama-cohort.md:8]: @Brandoncharleskhoo1 Any reason we link to LinkedIn here and not https://www.blitzllama.com/?

**Review by markzegarelli — APPROVED** [review]

File: docs/data/destinations/blitzllama-cohort.md (hunk 4 above)
Comment (Brandoncharleskhoo1) [reply docs/data/destinations/blitzllama-cohort.md]: Good catch! Fixed it :)

**Pull request merged**